modules/preview-react/index.ts

```diff
@@ -6,3 +6,5 @@ export * from './select';
 export * from './side-panel';
 export * from './text-area';
 export * from './text-input';
+
+/// <reference types="@types/node" />
```

Comment:
This removes Typescript errors complaining that `process` doesn't exist. `uuid` added `@types/node` to the global namespace for us.